src/seedu/addressbook/commands/Command.java

```diff
@@ -70,4 +72,9 @@ public int getTargetIndex() {
     public void setTargetIndex(int targetIndex) {
         this.targetIndex = targetIndex;
     }
+
+    public boolean isMutating(String userCommandText) {
```

Comment:
This method should have a header comment. If not, users of the method as well as those who overrides method will not know how exactly the method behaves. Also, as pointed out in the other PR, you should make use of polymorphism and implement the different behavior in individual sub classes